samples/NativeLibrary/c_source/loadlibrary.c

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would merge `loadlibrary.c` and `main.c` into a single file. I do not see the point to have them split into two files.

Reply:
Sure, definitely better.